demos/mqtt/mqtt_demo_plaintext/Makefile

```diff
@@ -1,11 +1,11 @@
 CC = gcc
 
 INCLUDE_DIRS = -I . \
-               -I ../../../libraries/standard/mqtt/include \
+               -I ../../../libraries/standard/coreMQTT/include \
```

Comment:
This is a good find. I don't think we need this file anymore. Do you know of a reason why we would need this file. This is out of the scope of this PR, but we may need to just remove it.

Reply:
https://github.com/aws/aws-iot-device-sdk-embedded-C/pull/1182
Had created a PR to remove this unused file once I found it in change set 😃